addons/website/static/src/js/widgets/dialog.js

```diff
@@ -0,0 +1,20 @@
+/** @odoo-module **/
+
+import Dialog from 'web.Dialog';
+
+Dialog.include({
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    _isBlocking(index, el) {
+        if (el.dataset['scrollBody']) {
```

Comment:
- It's ok but you are the tenth occurence of `.dataset['hello']` in Odoo code, in comparison to nearly a thousand of `.dataset.hello`. Maybe use the common form :wink: 
- Is that actually needed? Can't you exclude based on `#website_cookies_bar` checking the parent (advantage of not using a string selector like before)? The cookie view does not get updated for old clients already using it so it makes even more sense. That would also avoid a view change in stable. Also, is not the fix required before 15.0?

Reply:
Changed to checking the parent id instead, and removed the data parameter in the view template.

The change is not needed in 14.0 because the problem was introduced during the migration of web to owl in 15.0.